service/src/main/java/bio/terra/workspace/service/iam/model/SamConstants.java

```diff
@@ -8,16 +8,29 @@
   public static final String SAM_WORKSPACE_OWN_ACTION = "own";
   public static final String SAM_WORKSPACE_DELETE_ACTION = "delete";
   public static final String SAM_WORKSPACE_READ_IAM_ACTION = "read_policies";
-  public static final String SAM_CREATE_CONTROLLED_APPLICATION_SHARED =
+  public static final String SAM_CREATE_CONTROLLED_USER_SHARED_ACTION =
+      "create_controlled_user_shared";
+  public static final String SAM_CREATE_CONTROLLED_USER_PRIVATE_ACTION =
+      "create_controlled_user_private";
+  public static final String SAM_CREATE_CONTROLLED_APPLICATION_SHARED_ACTION =
       "create_controlled_application_shared";
-  public static final String SAM_CREATE_CONTROLLED_APPLICATION_PRIVATE =
+  public static final String SAM_CREATE_CONTROLLED_APPLICATION_PRIVATE_ACTION =
       "create_controlled_application_private";
   public static final String SAM_CREATE_REFERENCED_RESOURCE = "create_referenced_resource";
   public static final String SAM_UPDATE_REFERENCED_RESOURCE = "update_referenced_resource";
   public static final String SAM_DELETE_REFERENCED_RESOURCE = "delete_referenced_resource";
   public static final String SPEND_PROFILE_RESOURCE = "spend-profile";
   public static final String SPEND_PROFILE_LINK_ACTION = "link";
 
+  public static final String SAM_CONTROLLED_USER_SHARED_WORKSPACE_RESOURCE =
```

Comment:
Ah we might have added these constants in the `azure-poc` branch -- oh well we'll have to resolve merge conflicts.